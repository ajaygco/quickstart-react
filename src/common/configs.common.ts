type appConfig = {
  appName: string;
};

// App Config
export const appConfig = {
  appName: "Kickstart",
};
