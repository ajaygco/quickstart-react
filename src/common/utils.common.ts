// Get UTC Timestamp
export const getUtcTimestamp = (): number => {
  return new Date().valueOf();
};
