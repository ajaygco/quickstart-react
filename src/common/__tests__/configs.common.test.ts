// yarn test source/common/__tests__/configs.common.test.js

import { appConfig } from "@app/common/configs.common";

describe("configs.common", () => {
  describe("appConfig", () => {
    it("should match snapshot", () => {
      expect(appConfig).toMatchSnapshot();
    });
  });
});
