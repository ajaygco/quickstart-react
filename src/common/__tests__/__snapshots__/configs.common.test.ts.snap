// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`configs.common > appConfig > should match snapshot 1`] = `
{
  "appEmail": "team@app.com",
  "appName": "Kickstart",
  "copyrightYear": 2099,
  "developerName": "Your Company",
  "developerUrl": "https://yourcompany.co",
}
`;
